spec/support/common_helpers.rb

```diff
@@ -150,10 +150,10 @@ def transition(machine, state_event, user)
   # Closes a modal dialog and waits until it's really closed
   #
   def close_modal_dialog
-    within(".modal") do
+    within(".modal-diaglog") do
       click_link "Back"
     end
-    page.should_not have_css(".modal-backdrop", :wait => 15)
+    page.should_not have_css(".modal-backdrop", :wait => 30)
```

Comment:
Actually I was trying to fix the test but the problem is not here so I will revert this